_episodes/08-reproducibility.md

```diff
@@ -0,0 +1,144 @@
+
+  - The same computational work can be run across multiple different technologies seamlessly (e.g. Windows, macOS, Linux).
+  - You can save the exact process that you used for your computational work (rather than relying on potentially incomplete notes).
+  - You can save the exact versions of software and their dependencies in the image.
+  - You can access legacy versions of software and underlying dependencies which may not be generally available any more.
+  - You can archive and share the image as well as associating a persistent identifier with an image to allow other researchers to reproduce and build on your work.
+
+## Sharing images
+
+As we have already seen, the Docker Hub provides a platform for sharing images publicly. Once you have uploaded an image, you can point people to its public location and they can download and build upon it.
+
+This is fine for working collaboratively with images on a day-to-day basis but the Docker Hub is not a good option for long time archive of images in support of research and publications as:
+
+  - free accounts have a limit on how long an image will be hosted if it is not updated
+  - it does not support adding persistent identifiers to images
+  - it is easy to overwrite tagged images with newer versions by mistake.
+
+## Archiving and persistently identifying images using Zenodo
+
+When you publish your work or make it publicly available in some way it is good practice to make images that you used for computational work available in an immutable, persistent way and to have an identifier that allows people to cite and give you credit for the work you have done. [Zenodo](https://zenodo.org/) provides this functionality.
+
+Zenodo supports the archiving of *tar* archives and we can capture our Docker images as tar archives using the `docker save` command:
+
+~~~
+docker save image/name:tag -o image-archive.tar
+~~~
+{: .bash}
+
+These tar images can become quite large and Zenodo supports uploads up to 50GB so you may need to compress your archive to make it fit on Zenodo using a tool such as gzip (or zip):
+
+~~~
+gzip image-archive.tar
+~~~
+{: .bash}
+
+Once you have your archive, you can [deposit it on Zenodo](https://zenodo.org/deposit/) and this will:
+
+   - Create a long-term archive snapshot of your Docker image which people (including your future self) can download and reuse or reproduce your work.
+   - Create a persistent DOI (*Digital Object Identifier*) that you can cite in any publications or outputs to enable reproducibility and recognition of your work.
+
+In addition to the archive file itself, the deposit process will ask you to provide some basic metadata to classify the image and the associated work.
+
+Note that Zenodo is not the only option for archiving and generating persistent DOIs for images. There are other services out there - for example, some organizations may provide their own, equivalent, service.
```

Comment:
I love the addition of this section.  Should we write it for them to actually do this process with an image created earlier in the lesson?